templates/pharmacies/general_list.html

```diff
@@ -1,25 +1,25 @@
-<a href="controller.php?practice_settings&{$TOP_ACTION}pharmacy&action=edit" {php} if (!$GLOBALS['concurrent_layout']) echo "target='Main'"; {/php} onclick="top.restoreSession()" class="css_button" >
+<a href="controller.php?practice_settings&{$TOP_ACTION}pharmacy&action=edit" onclick="top.restoreSession()" class="css_button" >
 <span>{xl t='Add a Pharmacy'}</span></a><br><br>
 
 <table cellpadding="1" cellspacing='0' class="showborder">
-	<tr class="showborder_head">
-		<th width="170px"><b>{xl t='Name'}</b></th>
-		<th width="320px"><b>{xl t='Address'}</b></th>
-		<th><b>{xl t='Default Method'}</b></th>
-	</tr>
-	{foreach from=$pharmacies item=pharmacy}
-	<tr height="22">
-		<td><a href="{$CURRENT_ACTION}action=edit&id={$pharmacy->id}" onclick="top.restoreSession()">{$pharmacy->name}&nbsp;</a></td>
-		<td>
-		{if $pharmacy->address->line1 != ''}{$pharmacy->address->line1}, {/if}
-		{if $pharmacy->address->city != ''}{$pharmacy->address->city}, {/if}
-			{$pharmacy->address->state|upper} {$pharmacy->address->zip}&nbsp;</td>
-		<td>{$pharmacy->get_transmit_method_display()}&nbsp;
-	{foreachelse}</td>
-	</tr>
+    <tr class="showborder_head">
+        <th width="170px"><b>{xl t='Name'}</b></th>
+        <th width="320px"><b>{xl t='Address'}</b></th>
+        <th><b>{xl t='Default Method'}</b></th>
+    </tr>
+    {foreach from=$pharmacies item=pharmacy}
+    <tr height="22">
+        <td><a href="{$CURRENT_ACTION}action=edit&id={$pharmacy->id}" onclick="top.restoreSession()">{$pharmacy->name}&nbsp;</a></td>
+        <td>
+        {if $pharmacy->address->line1 != ''}{$pharmacy->address->line1}, {/if}
+        {if $pharmacy->address->city != ''}{$pharmacy->address->city}, {/if}
+            {$pharmacy->address->state|upper} {$pharmacy->address->zip}&nbsp;</td>
+        <td>{$pharmacy->get_transmit_method_display()}&nbsp;
+    {foreachelse}</td>
+    </tr>
 
-	<tr class="center_display">
-		<td colspan="3"><b>{xl t='No Pharmacies Found'}<b></td>
-	</tr>
-	{/foreach}
+    <tr class="center_display">
+        <td colspan="3"><b>{xl t='No Pharmacies Found'}<b></td>
+    </tr>
+    {/foreach}
 </table>
```

Comment:
New line please...if this is about removing tabs and formatting, might as well bring it to standard.